File: frontend/mgramseva/lib/model/reports/collection_report_data.dart
class CollectionReportData {
  String? tenantName;
  String? consumerName;
  String? connectionNo;
  String? oldConnectionNo;
  String? userId;
  String? paymentMode;
  double? paymentAmount;

  CollectionReportData(
      {this.tenantName,
        this.consumerName,
        this.connectionNo,
        this.oldConnectionNo,
        this.userId,
        this.paymentMode,
        this.paymentAmount});

  CollectionReportData.fromJson(Map<String, dynamic> json) {
    tenantName = json['tenantName'];
    consumerName = json['consumerName'];
    connectionNo = json['connectionNo'];
    oldConnectionNo = json['oldConnectionNo'];
    userId = json['userId'];
    paymentMode = json['paymentMode'];
    paymentAmount = json['paymentAmount'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['tenantName'] = this.tenantName;
    data['consumerName'] = this.consumerName;
    data['connectionNo'] = this.connectionNo;
    data['oldConnectionNo'] = this.oldConnectionNo;
    data['userId'] = this.userId;
    data['paymentMode'] = this.paymentMode;
    data['paymentAmount'] = this.paymentAmount;
    return data;
  }
}

File: municipal-services/ws-services/src/main/java/org/egov/waterconnection/web/models/CollectionReportData.java
package org.egov.waterconnection.web.models;

import java.math.BigDecimal;
import java.util.List;
import java.util.Map;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CollectionReportData {
	
	@JsonProperty("tenantName")
	private String tenantName = null;
	
	@JsonProperty("consumerName")
	private String consumerName = null;
	
	@JsonProperty("connectionNo")
	private String connectionNo = null;

	@JsonProperty("oldConnectionNo")
	private String oldConnectionNo = null;
	
	@JsonProperty("userId")
	private String userId = null;
	
	@JsonProperty("paymentMode")
	private String paymentMode = null;
	
	@JsonProperty("paymentAmount")
	private BigDecimal paymentAmount = null;
}
